### PR TITLE
Update aws docs for 0.6 release

### PR DESCRIPTION
**Pull request by Jeffwan**
Update aws provision process to use kfctl golang cli and kustomize

This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1030)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-1030--competent-brattain-de2d6d.netlify.com/docs/aws/

File: content/docs/aws/deploy/install-kubeflow.md [hunk 1]
```diff
@@ -19,136 +16,129 @@ deploy Kubeflow on Amazon Web Services (AWS).
...
 
+    * KFAPP - Use a relative directory name here rather than absolute path, such as `kfapp`. It will be used as eks cluster name.
+    * CONFIG - Path to the configuration file
 
-    * AWS_CLUSTER_NAME - Specify a unique name for your Amazon EKS cluster.
-    * KFAPP - Use a relative directory name here rather than absolute path, such as `kfapp`.
-    * REGION - Use the AWS Region you want to create your cluster in.
 
-1. Generate and apply platform changes.
+    __*Important!!!*__ By default, these scripts create an AWS Application Load Balancer for Kubeflow that is open to public. This is good for development testing and for short term use, but we do not recommend that you use this configuration for production workloads.
```
Comment (sarahmaddox) [inline content/docs/aws/deploy/install-kubeflow.md]: I think the bold text is enough and you can remove all the explanation marks. :)

File: content/docs/aws/deploy/install-kubeflow.md [hunk 2]
```diff
@@ -19,136 +16,129 @@ deploy Kubeflow on Amazon Web Services (AWS).
...
-    ```
 
-    > Note: If you see `no kind "ClusterConfig" is registered for version "eksctl.io/v1alpha5"` error, it means your eksctl version is low and not compatible with latest version. Please upgrade it and have another try.
+1. Wait for all the resources to become ready in the `kubeflow` namespace.
+    ```
+    kubectl -n kubeflow get all
+    ```
 
-1. Generate and apply the Kubernetes changes.
+1. Open Kubeflow Dashboard
```
Comment (sarahmaddox) [inline content/docs/aws/deploy/install-kubeflow.md]: Does the command below actually open the dashboard? The existing guide has more info here, and I think people will need more info to figure out how to make the Kubeflow UI appear.

**Comment by sarahmaddox** [comment]
Fixes https://github.com/kubeflow/website/issues/1024

File: content/docs/aws/deploy/install-kubeflow.md [hunk 3]
```diff
@@ -19,136 +16,129 @@ deploy Kubeflow on Amazon Web Services (AWS).
...
-
-* **platform** - All AWS resources; that is, anything that doesn’t run on Kubernetes. Like IAM policy attachments, Amazon EKS cluster creation, etc.
-* **k8s** - All Kubernetes resources. Such as Kubeflow packages and add-on packages like `fluentd` or `istio`.
-* **all** - Both AWS and Kubernetes resources.
-
-### App layout
+## Kubeflow installation
 
-Your Kubeflow `app` directory contains the following files and directories:
+1. Download the latest `kfctl` golang binary from [kubeflow release oage](https://github.com/kubeflow/kubeflow/releases) and unpack it.
```
[the remark this answers is not in the document]
Comment (Jeffwan) [reply content/docs/aws/deploy/install-kubeflow.md]: Good catch!

File: content/docs/aws/deploy/install-kubeflow.md (hunk 2 above)
Comment (Jeffwan) [reply content/docs/aws/deploy/install-kubeflow.md]: I change to get service endpoint and it make more sense now.

**Comment by Jeffwan** [comment]
/hold

Need to address indent to make markdown better rendered.

File: content/docs/aws/deploy/install-kubeflow.md (hunk 1 above)
Comment (Jeffwan) [reply content/docs/aws/deploy/install-kubeflow.md]: Make it

File: content/docs/aws/deploy/existing-cluster.md [hunk 4]
```diff
@@ -33,15 +31,22 @@ If you would like to deploy Kubeflow on existing Amazon EKS cluster, the only di
     eksctl-kubeflow-example-nodegroup-ng-185-NodeInstanceRole-1DDJJXQBG9EM6
     ```
 
-    If you have multiple node groups, you will see corresponding number of node group roles. In that case, please provide the role names as a comma-separated list.
+    If you have multiple node groups, you will see corresponding number of node group roles. In that case, please provide the role names as an array.
+
+    ```shell
```
[the remark this answers is not in the document]
Comment (Jeffwan) [reply content/docs/aws/deploy/existing-cluster.md]: Looks like my local rendered website looks a little bit different from this server. Not sure if that's the issue of different hugo server and markdown render engine problem. 

Seems I need to remove `shell` in code blocks to make sure unorder list work well.

[the remark this answers is not in the document]
Comment (Jeffwan) [reply content/docs/aws/deploy/existing-cluster.md]: I have to use 8 space before `- eksctlxxx` but this breaks yaml format. If I use 6 spaces (4 spaces + yaml 2 spaces), engine will render it as ordered list. Sarah, do you have any idea of how to fix it?

[the remark this answers is not in the document]
Comment (Jeffwan) [reply content/docs/aws/deploy/existing-cluster.md]: Get this fixed by giving 2 more space for codes

**Comment by Jeffwan** [comment]
@sarahmaddox Please have a final review

**Comment by sarahmaddox** [comment]
Thanks!
/lgtm
/approve

**Pull request merged**